### DISCUSSION TITLE
tidak bisa buat environment baru dengan python version 3.10.8

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#610] [Q&A] [answered]
saya ingin buat env baru dengan python version 3.10.8 akan tetapi mendapatkan error

![image](https://user-images.githubusercontent.com/23212456/229709853-fb322f7c-6e4f-4567-a6f6-8a63a60b1ce9.png)

**Comment by kevbow** [comment, marked as answer]
Dear @Gustana,

Terdapat kesalahan pada syntax yang Anda tuliskan, seharusnya menggunakan:
```
conda create -n "nama_environment" python=3.10.8
```
Demikian dan terima kasih.
